DUI3-DX/DUI3/Speckle.Connectors.DUI/Bridge/BrowserBridge.cs

```diff
@@ -251,11 +250,15 @@ private void ExecuteMethod(string methodName, string requestId, string args)
   /// </summary>
   /// <param name="requestId"></param>
   /// <param name="serializedData"></param>
-  private void NotifyUIMethodCallResultReady(string requestId, string serializedData = null)
+  private void NotifyUIMethodCallResultReady(string requestId, string? serializedData = null)
   {
-    _resultsStore[requestId] = serializedData;
+    if (serializedData is not null)
+    {
+      _resultsStore[requestId] = serializedData;
```

Comment:
perhaps the `_resultsStore` should be changed to allow nullable values. I think we're expecting `null` to be a valid result